### PR TITLE
README written in french and english

### PR DESCRIPTION
**Pull request by ilalande**

### CODE REVIEW

**Review by ThomasDCW — APPROVED** [review]
ok for me

File: LISEZ-MOI.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+## Table of Contents
 
-## Installation & Utilisation
+1. [General Info](#general-info)
+2. [Technologies](#technologies)
+3. [Installation](#installation)
+4. [Auteurs](#auteurs)
 
-### Pour commencer un projet
+## General Info
```
Comment (LuigiMenez) [inline LISEZ-MOI.md:10]: Why it's in English?

**Review by LuigiMenez — APPROVED** [review]
Ok for me?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,79 @@
...
+- _Vite_ : Alternative to _Create-React-App_, with fewer packages for a smoother experience
+- _ESLint_ : "Code quality" tool, ensures that pre-configured rules are respected
+- _Prettier_ : "code quality" tool, also focuses on code style
+- _Standard Airbnb_ : One of the most famous "standards", even if it is not officially linked to ES/JS
+- _Nodemon_ : Tool to restart a server each time one of the files is modified
+
+### Other libraries
+
+- MySQL: A database has been developed on mysql thanks to WorkBench
+- ReactAdmin` : Course and user management has been developed on React Admin
```
Comment (LoicBrassart) [inline README.md:39]: Broken line

File: README.md [hunk 3]
```diff
@@ -1,34 +1,79 @@
...
+## Team
+
+A team of developers in training from March to June 2022 at the Wild Code School in Lille
+
+- Amar
+- Faycel
+- Rémi
+- Jérémy
+- Juliane
+- Thomas
```
Comment (LoicBrassart) [inline README.md:79]: Add links ?

File: LISEZ-MOI.md [hunk 4]
```diff
@@ -32,18 +32,47 @@ Ce template est conçu pour servir de base à tous les projets (P2/P3) suivants
...
 
-- s'assurer que les principaux outils utilisés lors de la formation sont compatibles avec ce template
-- deploiements ? Compatible avec Netlify/Vercel/Heroku ?
-- fonctionnement avec yarn/pnpm
+- Amar
+- Faycel
+- Rémi
+- Jérémy
+- Juliane
+- Thomas
```
Comment (LoicBrassart) [inline LISEZ-MOI.md:78]: Add links ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ilalande) [reply README.md:39]: I don't understand "broken line" : it means there is a blank line I must delete? (I put one after every title) or something else? (in which case I don't understand what)

**Review by ThomasDCW — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**